release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -1860,6 +1860,8 @@ fail to boot. Do not select this option.
 * You might receive a message that the system is unable to mount a volume when the volume contains a large number of files. This can happen when a Pod mounts a volume that is set with `FSGroup SecurityContext` because the GID ownership of the files must be recursively updated for all files on the volume. Users should expect that Pods using volumes with a large number of files and the `FSGroup SecurityContext` setting can take considerable time to start.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1515907[*BZ#1515907*])
 
+* On Azure when upgrading from 4.4 to 4.5, the Ingress Operator fails to ensure a DNSRecord due to errors refreshing the token. Restarting the Ingress Operator fixes the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1854383[*BZ#1854383*])
```

Comment:
`the Ingress Operator might fails...` since it happens randomly and rarely.

Reply:
other looks good. thank you @bmcelvee

Reply:
Thank you!